site/InteractionNotice.tsx

```diff
@@ -1,16 +0,0 @@
-export const INTERACTIVE_ICON_SVG = `<svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="hand-pointer" class="svg-inline--fa fa-hand-pointer fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 617">
```

Comment:
Deleting `InteractionNotice` because we weren't using it.

`shouldProgressiveEmbed` has been hard-coded to true for the last year, so all mobile devices have been embedding charts normally.

Which means that the only time this _would_ render is if JS was disabled, but we were _hiding_ the notice when JS was disabled because it doesn't make sense to explain to the user that clicking on the thumbnail will load an interactive chart in that case 🙃 

We could have rewritten `shouldProgressiveEmbed`, but I think we can circle back to that if/once https://github.com/owid/owid-grapher/pull/4377 is done.